src/styles/_typography.scss

```diff
@@ -1,3 +1,3 @@
-h1 {
-  @extend %h1;
-}
+// h1 {
```

Comment:
The h1 styling has been commented out, which may cause unintended styles to be applied to h1 elements on the landing page. If you want to use the %h1 style for h1 elements, uncomment the lines.